src/lazy_sockets.h

```diff
@@ -203,6 +203,22 @@ class LSocket {
 		#endif // #ifdef WIN
 	}
 
+	// receives a :size: bytes into :buffer:
+	// returns the number of received bytes or -1 on error
+	inline int RecvFrom(void* buff, size_t size, lcsockaddr_in& sender, size_t& sender_size, ERecvFlags flags=ERecv_none) {
+		// compile time check for windows stupidity
+		#ifdef WIN
+		int send_size = sender_size;
+		int recv = recvfrom(_soc_handle, (char*)buff, (int)size, flags, (sockaddr *)&sender, &send_size);
+		sender_size = send_size;
+		return recv;
+		#elif defined(LINUX)
+		return recv(_soc_handle, buff, size, flags);
```

Comment:
this is not recvfrom, see [`man 3 recvfrom`](https://linux.die.net/man/3/recvfrom)

Reply:
accepts pretty much the same args as windows version

Reply:
fixed